rules/exfil/stealer/wallet.yara

```diff
@@ -30,6 +30,9 @@ rule crypto_stealer_names: critical {
     $not_js          = /\"js\": \{[^}]{0,64}/
     $not_scriptsrc   = /\"scriptSrc\": "([^"]{0,64})"/
     $not_website     = /\"website\": "([^"]{0,64})"/
+    $not_geth_mod    = "github.com/ethereum/go-ethereum"
+    $not_clef        = "github.com/ethereum/go-ethereum/cmd/clef/main.go"
```

Comment:
The cmd rules are covered by `$not_geth_mod`; but it brings up an interesting problem that we'd be invisible to any stealer that imports the module. That's probably OK though, we can make more specific rules for those cases.

Reply:
I'll see if I can find a better string for these.

Reply:
Updated in [`94f75d4` (#838)](https://github.com/chainguard-dev/malcontent/pull/838/commits/94f75d4a846047ff3d98be23b379eb9d577c6407).

I avoided any import-esque strings.